dat/missions/neutral/pirbounty_dead.lua

```diff
@@ -264,6 +273,7 @@ function pilot_death( p, attacker )
    if attacker == player.pilot() then
       succeed()
       target_killed = true
+      misn.appendLog("Successfully killed the pirate")
```

Comment:
Assuming the log is supposed to be a useful reference for important past events, this is just cluttering it up with unnecessary information.